tests/cupyx_tests/scipy_tests/signal_tests/test_signaltools.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+import pytest
+import scipy.signal  # NOQA
+
+import cupy
+import cupyx
+from cupy import testing
+
+
+@testing.gpu
+@testing.parameterize(*testing.product({
+    'mode': ['valid', 'same', 'full'],
+    'shape': [((), (5,)), ((3, 4, 5), (1, 2))]
```

Comment:
This parameter is used only for `test_choose_conv_method_ndim`. Please split the test class.